dev/com.ibm.ws.security.saml.sso_fat.common/SAMLCommon.gradle

```diff
@@ -29,8 +29,6 @@ dependencies {
    */
   requiredLibs project(':com.ibm.ws.security.saml.sso_fat.common'),
                project(':com.ibm.ws.org.apache.httpcomponents'),
-               'commons-codec:commons-codec:1.4',
-               'commons-logging:commons-logging:1.1.1',
                'httpunit:httpunit:1.6.2',                           // 1.7 has issues noted above
```

Comment:
I guess this project just didn't really require these libraries at all?

Reply:
hmmm i'm not sure what happened there ... should have just been switched to the new projects